ujson.go

```diff
@@ -6,17 +6,20 @@ import (
 )
 
 type JSON struct {
-	root interface{}
+	Root interface{}
```

Comment:
I would prefer to not expose this.

Alternatively, if `Array()` returned `[]*ujson.JSON` then the above code would be even easier.